baseplate/server/__init__.py

```diff
@@ -237,11 +238,45 @@ def load_and_run_script():
     parser.add_argument(
         "entrypoint", type=_load_factory, help="function to call, e.g. module.path:fn_name"
     )
+    parser.add_argument(
```

Comment:
i wonder if it'd be more error-message-friendly to `parse_known_args` then determine if the entrypoint accepts more args and re-parsing accordingly. :no_bell: